drivers/lvutil.py

```diff
@@ -529,6 +529,22 @@ def createVG(root, vgname):
 
     # End block
 
+def getPVsInVG(vgname):
+    # Get PVs in a specific VG
+    pvs_ret = cmd_lvm([CMD_PVS, '--separator', ' ', '--noheadings', '-o', 'pv_name,vg_name'])
+    
+    # Parse each line to extract PV and VG information
+    # No need to handle exceptions here, return empty list if any error
+    pvs_in_vg = []
+    lines = pvs_ret.strip().split('\n')
+    for line in lines:
+        pv, vg = line.split()
```

Comment:
Can we guarantee that the output at _line:534_ consistently matches the pattern `/dev/pv_xxx vg_xxx ` in all scenarios? If not, here might be exceptions.

Reply:
The '-o' option in the command at line 534 specifies the fields to display.

Reply:
I random selected a host and tested , and the output was: `/dev/sda3--noheadingsVG_XenStorage-6d5630c0-a016-b2b8-d5fa-e8faed85ca3a`

Reply:
There is a space ' ' before --noheadings.